barretenberg/cpp/src/barretenberg/stdlib/primitives/field/field.hpp

```diff
@@ -129,6 +129,9 @@ template <typename Builder> class field_t {
     // N.B. we implicitly range-constrain 'exponent' to be a 32-bit integer!
     field_t pow(const field_t& exponent) const;
```

Comment:
this one maybe shouldn't allow for implicit size_t to field_t conversions

Reply:
I don't think there's any problem with passing this a size_t in general. field_t has constructors from all kinds on int types. It just has to be used correctly for witnesses